.github/workflows/ready_for_review.yml

```diff
@@ -181,22 +218,12 @@ jobs:
           fi
 
       - name: Add ready-for-backend-review label
+        if: steps.verify_approval.outputs.approval_status == 'required'
         uses: actions-ecosystem/action-add-labels@v1
-        if: |
-          steps.audit_pr_labels.outputs.failures_detected == 'false' &&
-          steps.verify_approval.outputs.approval_status == 'required'
         with:
           number: ${{ env.pr_number }}
           labels: ready-for-backend-review
 
-      - name: Remove ready-for-backend-review label
-        uses: actions-ecosystem/action-remove-labels@v1
-        if: |
-          ( 
-            steps.audit_pr_labels.outputs.failures_detected == 'true' ||
-            steps.verify_approval.outputs.approval_status == 'confirmed'
-          ) &&
-          contains(env.pr_labels, 'ready-for-backend-review')
-        with:
-          number: ${{ env.pr_number }}
-          labels: ready-for-backend-review
+
```

Comment:
The require backend approval workflow will remove this. no need to remove this twice